package.json

```diff
@@ -265,7 +268,7 @@
     "mockery": "1.7.0",
     "nock": "8.0.0",
     "nodemon": "1.4.1",
-    "prettier": "Automattic/calypso-prettier#0cbd77dcfc7bb82bff5f4485a6f078dec5cc9595",
+    "prettier": "github:automattic/calypso-prettier#70f962b0",
```

Comment:
why aren't we using releases here?

Reply:
We used to have `.../calypso-prettier#calypso-1.6` here -- mentioning the branch by name instead of commit SHA. But that lead to random upgrades on every shrinkwrap update, as the `calypso-1.x` branch was evolving. So we switched to a hard commit reference.

We could start doing releases in the `calypso-prettier` repo, but we'd have to invent a versioning scheme that's different from vanilla semver.

The fork is based on an upstream version (e.g., 1.6.1) and that should be a part of the fork version number.

And then we can have multiple bugfix releases of the fork itself -- leading to versions like 1.6.1-1, 1.6.1-2, ...